.buckconfig

```diff
@@ -8,6 +11,15 @@
   # force usage of oss_shim.bzl which applies all the correct visibility logic
   # and more closely mimics the internal macros
   disable_implicit_native_rules = true
+  # This enables buck to switch to the host platform for binaries
+  # that are used in $(exe) macros, since they are likely being executed
+  # against the host platform
+  host_configuration_switch_enabled = true
+
+  # The default target platform is the host platform.  This ensures
+  # that targets that aren't immediate depends of an image_layer
+  # are built with the host platform
+  target_platforms=config//platform:linux-x86_64
```

Comment:
This particular buck config is on the deprecation path. Instead, it can be replaced by `parser.target_platform_detector_spec`.  That equivalent is

```
    target_platform_detector_spec = target://...->config//platform:linux-x86_64
```

I'm not sure the timeline for deprecation (and your case actually gives some reason for why it should be kept).

Reply:
This will be a static config since we never want to tweak the host platform.  I'll switch it to the detector_spec.